docs/_static/auth/v2.yaml

```diff
@@ -0,0 +1,247 @@
+openapi: 3.0.0
+info:
+  title: Auth v2 API (experimental)
+  version: 0.0.0
+servers:
+  - url: "http://localhost:1323"
+paths:
+  "/iam/{did}/token":
```

Comment:
comment: public paths need to be discussed later

Reply:
done